plugin/src/test/java/io/jenkins/plugins/casc/ConfigurationAsCodeTest.java

```diff
@@ -163,4 +166,20 @@ public void doViewExport_should_require_authentication() throws Exception {
         response = loggedInClient.loadWebResponse(request);
         assertThat(response.getStatusCode(), is(200));
     }
+
+    @Test
+    @Issue("Issue #739")
+    public void preferEnvOverGlobalConfigForConfigPath() throws Exception {
+        String firstConfig = getClass().getResource("JenkinsConfigTest.yml").toExternalForm();
+        String secondConfig = getClass().getResource("merge3.yml").toExternalForm();
+        CasCGlobalConfig descriptor = (CasCGlobalConfig) j.jenkins.getDescriptor(CasCGlobalConfig.class);
+        assertNotNull(descriptor);
+        descriptor.setConfigurationPath(firstConfig);
+        ConfigurationAsCode.get().configure();
+        Assert.assertThat(j.jenkins.getDescription(), is("configuration as code - JenkinsConfigTest"));
+        System.setProperty(CASC_JENKINS_CONFIG_PROPERTY, secondConfig);
+        ConfigurationAsCode.get().configure();
+        Assert.assertThat(j.jenkins.getDescription(), is("Configured by Configuration as Code plugin"));
+        System.clearProperty(CASC_JENKINS_CONFIG_PROPERTY);
```

Comment:
Just a minor hint: In case the test case fails before this line, but after line 180 where the system property is set, then it is still set and potentially causing side effects in other test cases... (cf. https://stefanbirkner.github.io/system-rules/ for JUnit rules reseting system properties -- maybe there is something like this also already in jenkins? or not so elegant, but easier with a simple try-finally usage)

Reply:
We already use system-rules I just wanted a release going we can always optimize.
Please send a PR fixing the issue.